packages/webdriverio/src/utils/refetchElement.js

```diff
@@ -0,0 +1,27 @@
+export default function refetchElement () {
+    let currentElement = this;
+
+    return new Promise ( async () => {
```

Comment:
why this way? Just do `export default async function refetchElement () {` and you can use `await` right away

Reply:
Updated.